### DISCUSSION TITLE
fail2ban not adding iptables Chain f2b-sshd

### DISCUSSION THREAD

**Discussion by unblog** [fail2ban/fail2ban#3074] [Q&A]
I have set up fail2ban in many times, but here it does not work with the newly installed Debian 10 buster, on other VMs with CentOS 7 and also a Debian 10 with `iptables -vnL` the chain `f2b-sshd` appears, the configuration jail.local are completely identical, ufw (Uncomplicated Firewall ) is enabled on both debian VMs. Default banning action is `banaction = iptables-multiport` defined in jail.local. I think it's something completely different, but I can't figure it out!

**Comment by sebres** [comment]
Since fail2ban 0.10 (IPv6 support) fail2ban executes `actionstart` IP-family related on demand by first ban per jail, see #1742, so `iptables-multiport` would create the chain `f2b-sshd` only if first IP gets banned in sshd jail.
